### ISSUE TITLE
iOS - VoiceOver Reading File Name for DOL Seal

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#90]
While grabbing a screenshot of the home page before choosing a language VoiceOver read the DOL seal as it's file name when it should be "United States Department of Labor seal".

There might be the same issue for the Android and web version of the page.

Please see attached screenshot highlighting the DOL seal.

![comply chain - ios - english - dol seal is reading file name instead of dol seal](https://user-images.githubusercontent.com/10423487/61413943-beccff80-a8ba-11e9-8342-93555955f132.png)

**Comment by jfc3-dol** [comment]
I am closing the issue because VoiceOver is now reading the DOL seal as  "United States Department of Labor seal" instead of the file name.
